### DISCUSSION TITLE
owncast 0.1.1 macOS

### DISCUSSION THREAD

**Discussion by MichaelSlomma** [owncast/owncast#3284] [Q&A]
I'm using the curl install script for installation and it seems to finish normally.

Starting owncast on a M1 machine (macOS 14):
The application hangs, no output, no log, nothing.
The process is running but doesn't do anything.

Startting owncast on an Intel machine (macOS 13):
I got a segmentation fault. Also no further output or logs.

The same happens if i download the release manually.

Any guesses?

**Comment by gabek** [comment]
Duplicate of #3056. Read more background there.
